sormas-backend/src/test/java/de/symeda/sormas/backend/externalmessage/ExternalMessageFacadeEjbMappingTest.java

```diff
@@ -56,6 +60,7 @@ public void testFromDto() {
 		assignee.setUuid("12345");
 
 		when(sampleReportFacade.fromDto(eq(sampleReportDto), any(ExternalMessage.class), eq(false))).thenReturn(sampleReport);
+		when(countryService.getByReferenceDto(source.getPersonCountry())).thenReturn(null);
```

Comment:
would be more useful to mock also the country than assert that the right country has been mapped to dto